crossdock/server/echo.go

```diff
@@ -0,0 +1,21 @@
+package server
+
+import (
+	"github.com/yarpc/yarpc-go"
+	"golang.org/x/net/context"
+)
+
+// EchoRequest contains a message to echo
+type EchoRequest struct {
+	Token string `json:"token"`
+}
+
+// EchoResponse contains a messaged echoed by EchoRequest
+type EchoResponse struct {
+	Token string `json:"token"`
```

Comment:
Try not to duplicate these types between the server and the client. Put them in a shared package somewhere.